fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -163,11 +164,12 @@ protected BDBEnvironment(File envHome, String selfNodeName, String selfNodeHostP
     // The setup() method opens the environment and database
     protected void setup() throws JournalException, InterruptedException {
         this.closing = false;
-        initConfigs();
+        ensureHelperInLocal();
+        initConfigs(isElectable);
```

Comment:
why pass the `isElectable` parameter?
This `isElectable` is not changed, you can use the member variable.